packages/hardhat-core/src/internal/cli/cli.ts

```diff
@@ -60,6 +64,38 @@ To learn more about which versions of Node.js are supported go to https://hardha
   }
 }
 
+async function suggestInstallingHardhatVscode() {
+  const alreadyPrompted = hasPromptedForHHVSCode();
+  if (alreadyPrompted) {
+    return;
+  }
+
+  const isInstalled = isHardhatVSCodeInstalled();
+
+  if (isInstalled !== InstallationState.EXTENSION_NOT_INSTALLED) {
+    return;
+  }
+
+  const installationConsent = await confirmHHVSCodeInstallation();
+  writePromptedForHHVSCode();
+
+  if (installationConsent === true) {
+    const installed = installHardhatVSCode();
```

Comment:
I would add a log here saying something like "Installing the extension..."

Reply:
Added that log